src/pages/terraform/docs/[...page].tsx

```diff
@@ -0,0 +1,10 @@
+import DocsView from 'views/docs-view'
+import { getRootDocsPathGenerationFunctions } from 'views/docs-view/utils/get-root-docs-path-generation-functions'
+
+const { getStaticPaths, getStaticProps } = getRootDocsPathGenerationFunctions(
```

Comment:
Note that `getRootDocsPathGenerationFunctions` works with both `[...page].tsx` and `[[...page]].tsx` files.

In this case, I've moved `docs` to a `[...page].tsx` file, since the landing page is handled by `src/pages/[productSlug]/docs/index.tsx`.